Makefile

```diff
@@ -305,35 +229,38 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-### install_cert_manager: install Cert Mananger v1.0.4 on the cluster
-install_cert_manager:
-	kubectl apply --validate=false -f https://github.com/jetstack/cert-manager/releases/download/v1.0.4/cert-manager.yaml
 
-_operator_sdk:
-	@{ \
-		if ! command -v operator-sdk &> /dev/null; then \
-			echo 'operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but it is not installed' ;\
-			exit 1 ;\
-		else \
-			SDK_VER=$$(operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) && \
-			if [ "$${SDK_VER}" != $(OPERATOR_SDK_VERSION) ]; then \
-				echo "WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $${SDK_VER} found" \
-				echo "WARN: Please use the recommended operator-sdk if you face any issue" \
-			fi \
-		fi \
-	}
-ifneq ($(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2),$(OPERATOR_SDK_VERSION))
-	@echo 'WARN: operator-sdk $(OPERATOR_SDK_VERSION) is expected to be used for this target but $(shell operator-sdk version | cut -d , -f 1 | cut -d : -f 2 | cut -d \" -f 2) found.'
-	@echo 'WARN: Please use the recommended operator-sdk if you face any issue.'
-endif
-
-_check_cert_manager:
-ifeq ($(PLATFORM),kubernetes)
-	if ! ${K8S_CLI} api-versions | grep -q '^cert-manager.io/v1$$' ; then \
-		echo "Cert-manager is required for deploying on Kubernetes. See 'make install_cert_manager'" ;\
-		exit 1 ;\
+.PHONY: compile-devworkspace-controller
+compile-devworkspace-controller:
```

Comment:
Could you add some doc lines according to the hacked-together `help:` rule we have?
```suggestion
.PHONY: compile-devworkspace-controller
### compile-devworkspace-controller: compile the devworkspace-controller binary
compile-devworkspace-controller:
```

Reply:
Yes! Sorry! I've added this!